db/structure.sql

```diff
@@ -9,13 +9,6 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
---
--- Name: public; Type: SCHEMA; Schema: -; Owner: -
---
-
```

Comment:
@PriyangaPKini this deletion has your git comment. Is it safe to delete?

Reply:
It's just a comment. It's fine